interface/patient_file/report/custom_report.php

```diff
@@ -844,7 +844,7 @@ function next_prev(action){
                   else {
                     echo "<img src='" . $GLOBALS['webroot'] .
                       "/controller.php?document&retrieve&patient_id=&document_id=" .
-                      $document_id . "&as_file=false'><br><br>";
+                      $document_id . "&as_file=false&original_file=true&disable_exit=false&show_original=true'><br><br>";
```

Comment:
You should not need to set the following here since you are just setting what is already the default settings:
original_file
original_file
disable_exit